sources/packages/web/src/components/generic/HeaderTitleValue.vue

```diff
@@ -0,0 +1,26 @@
+<!--
+Used when we need to display header title and the value
+-->
+<template>
+  <div>
+    <span class="label-bold-normal">{{ title }} : </span>
```

Comment:
Is there an additional space before the ":"?

Reply:
yes, it does